### DISCUSSION TITLE
Why RTK query cant give your functional for pass notifications about success or failed request

### DISCUSSION THREAD

**Discussion by KirillYoYo** [reduxjs/redux-toolkit#4196] [Ideas]
someone like this:

> createProfile: build.mutation<Profile, Omit<Profile, 'id'>>({
        query: (profile) => ({
          url: '/resources/profiles',
          method: 'POST',
          body: profile,
        }),
        invalidatesTags: ['Profile'],
        **SUCCESS_MESSAGE: 'bla bla success',
        FAILED_MESSAGE: 'request is failed....'**
    }),

**Comment by markerikson** [comment]
Sorry, this is a very unclear request.
